jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/AttributesTest.java

```diff
@@ -135,4 +139,88 @@ public void testAttributeLayer()
         testAttributes(layer);
     }
 
+    @ParameterizedTest
+    @MethodSource("attributes")
+    public void testSynthetic(Attributes attributes)
+    {
+        Assumptions.assumeTrue(attributes instanceof Attributes.Synthetic);
```

Comment:
Just get rid of the parametrization, allocate `TestSynthetic` and test only that.